nodeJS/introduction_to_nodeJS/introduction_what_is_nodeJS.md

```diff
@@ -35,20 +36,20 @@ For example, let's say you are writing a program and you need it to do the follo
 
-While both of these processes are running, Node sits and waits on an _event_. In this case, it is waiting on the completion of both processes, the reading of a file and the database query. When either of these tasks are finished, Node will fire off an event that will run the next function we've defined. So if the read-file process finishes first, it will print the file contents. If the database query finishes first, it will start the filtering process. As the programmer, we don't know or care which order the two processes are going to be completed. If this code was processed synchronously (rather than asynchronously) we would have to wait for each step in the program before moving on to the next one, which could cause things to slow down considerably. If the file that we needed to read was really long then we might have to wait a few seconds before the database query could begin.
+While both of these processes are running, Node sits and waits on an *event*. In this case, it is waiting on the completion of both processes, the reading of a file and the database query. When either of these tasks are finished, Node will fire off an event that will run the next function we've defined. So if the read-file process finishes first, it will print the file contents. If the database query finishes first, it will start the filtering process. As the programmer, we don't know or care which order the two processes are going to be completed. If this code was processed synchronously (rather than asynchronously) we would have to wait for each step in the program before moving on to the next one, which could cause things to slow down considerably. If the file that we needed to read was really long then we might have to wait a few seconds before the database query could begin.
 
-This process is almost exactly like the way that you would use `addEventListener` in front-end JavaScript to wait for a user action such as a mouse-click or keyboard press. The main difference is that the events are going to be things such as network requests and database queries. This functionality is facilitated through the use of callbacks. Callbacks are incredibly important to Node, so take a minute to read through [this article](https://dev.to/i3uckwheat/understanding-callbacks-2o9e) to make sure you're up to speed.
+This process is almost exactly like the way that you would use `addEventListener` in front-end JavaScript to wait for a user action such as a mouse-click or keyboard press. The main difference is that the events are going to be things such as network requests and database queries. This functionality is facilitated through the use of callbacks. Callbacks are incredibly important to Node, so take a minute to read through this [article](https://dev.to/i3uckwheat/understanding-callbacks-2o9e) to make sure you're up to speed.
```

Comment:
```suggestion
This process is almost exactly like the way that you would use `addEventListener` in front-end JavaScript to wait for a user action such as a mouse-click or keyboard press. The main difference is that the events are going to be things such as network requests and database queries. This functionality is facilitated through the use of callbacks. Callbacks are incredibly important to Node, so take a minute to read through [this article about understanding callbacks](https://dev.to/i3uckwheat/understanding-callbacks-2o9e) to make sure you're up to speed.
```

We could make this link more descriptive while we're at it. If you agree, you can either locally edit and commit or click `commit suggestion` for the same effect

Reply:
I agree. But to avoid lint issues, we need to keep ‘this’ out of the square brackets.

Reply:
Good call, I have been out of the loop for a few months and missed some of the Linter updates. Thanks for that, feel free to edit.

Reply:
I updated it so you can check it if you like :)